index.js

```diff
@@ -198,8 +209,7 @@ Toolkit.run(
       try {
         await commitFile();
       } catch (err) {
-        tools.log.debug("Something went wrong");
-        return tools.exit.failure(err);
+        return tools.exit.failure(err.message);
```

Comment:
Don't we want to keep the error code in the failure output?

Reply:
Yup, good to have; made the promise not to resolve with a value since it isn't consumed.